Sources/Map.swift

```diff
@@ -134,7 +134,7 @@ public final class Map {
 				#if swift(>=4.1)
 				return v.compactMap{ Float($0) } as? T
 				#else
-				return v.flatMap{ Float($0) } as? T
+				return v.compactMap{ Float($0) } as? T
```

Comment:
**Same comment as above.**

Please just remove `else` statement if it is necessary because now `if` and `else` blocks has same actions.